tests/unittests/OnnxImporterTest.cpp

```diff
@@ -199,6 +199,71 @@ TEST(onnx, importUniBroadcastMultiOutput) {
   (void)onnxLD;
 }
 
+static void testImportPRelu(std::string filename,
```

Comment:
I think there are a few more possible cases to cover with tests (i.e. different shapes of slope) that we may want to add in the future - this is why this can be useful.